en/docs/use-cases/streaming-tutorials/working-with-kafka.md

```diff
@@ -6,42 +6,40 @@ The Streaming Integrator can consume from a Kafka topic as well as to publish to
 
 This tutorial takes you through consuming from a Kafka topic, processing the messages, and finally, publishing output to a Kafka topic.
 
-!!!tip "Before you begin:"
-    Prepare the server to consume from or to publish to Kafka, follow the steps below:<br/>
-    <br/>
-        1. Download the Kafka broker from [the Apache site](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.3.0/kafka_2.12-2.3.0.tgz) and extract it.
-        From here onwards, this directory is referred to as `<KAFKA_HOME>`. <br/>
-        <br/>
-        2. Create a directory named `Source` in a preferred location in your machine and copy the following JARs to it from the `<KAFKA_HOME>/libs` directory.<br/>
-        <br/>
-            - `kafka_2.12-2.3.0.jar`<br/>
-            <br/>
-            - `kafka-clients-2.3.0.jar`<br/>
-            <br/>
-            - `metrics-core-2.2.0.jar`<br/>
-            <br/>
-            - `scala-library-2.12.8.jar`<br/>
-            <br/>
-            - `zkclient-0.11.jar`<br/>
-            <br/>
-            - `zookeeper-3.4.14.jar`<br/>
-            <br/>
-        3. Create another directory named `Destination` in a preferred location in your machine.<br/>
-        <br/>
-        4. To convert the Kafka JARS you copied to the `Source` directory, issue the following command:<br/>
-           ```
-           sh <SI_HOME>/bin/jartobundle.sh <{Source}_Directory_Path> <{Destination}_Directory_Path>
-           ```<br/>
-           <br/>
-        5. Copy all the jars from the `Destination` directory to the `<SI_HOME>/lib` directory.
-
-
-
-## Tutorial steps
-
-### Consuming data from Kafka
-
-### Start Kafka
+## Before you begin
+
+Prepare the server to consume from or to publish to Kafka, follow the steps below:
+
+1. Download the Kafka broker from [the Apache site](https://www.apache.org/dyn/closer.cgi?path=/kafka/2.3.0/kafka_2.12-2.3.0.tgz) and extract it. From here onwards, this directory is referred to as `<KAFKA_HOME>`.
+
+2. Create a directory named `Source` in a preferred location in your machine and copy the following JARs to it from the `<KAFKA_HOME>/libs` directory.
+
+    - `kafka_2.12-2.3.0.jar`
+   
+    - `kafka-clients-2.3.0.jar`
+    
+    - `metrics-core-2.2.0.jar`
+    
+    - `scala-library-2.12.8.jar`
+    
+    - `zkclient-0.11.jar`
+   
+    - `zookeeper-3.4.14.jar`
+  
+3. Create another directory named `Destination` in a preferred location in your machine.
```

Comment:
Shouldn't "Destination" be "destination" as it is a directory name. Please check this in other places like line 31

Reply:
Changed as specified